### PR TITLE
fix documentation for consistency and readability

### PR DESCRIPTION
**Pull request by waldyrious**
This should be fairly uncontroversial, but putting it out as a PR in case anyone has comments.

### CODE REVIEW

File: Wikimate.php [hunk 1]
```diff
@@ -196,8 +205,9 @@ public function query( $array ) {
 
 	/**
 	 * Performs a parse query to the wiki API.
-	 * @param array $array array of details to be passed in the query.
-	 * @return array unserialized php output from the wiki API.
+	 *
+	 * @param   array  $array  Array of details to be passed in the query.
```
Comment (Xymph) [inline Wikimate.php]: All other param lines have no dot at the end - trim them in these two lines too?

File: Wikimate.php [hunk 2]
```diff
@@ -356,47 +372,52 @@ function destroy() {
...
+	 * Getter functions
 	 *
 	 */
 	
 	/**
-	 * Gets the text of the page.  If refesh is true then this method will
-	 * query the wiki api again for the page details
-	 * @param boolean $refresh true to query the wiki api again
-	 * @return string the text of the page
+	 * Gets the text of the page. If refesh is true,
```
Comment (Xymph) [inline Wikimate.php]: Please fix the old typo in refesh while you're at it.

File: Wikimate.php [hunk 3]
```diff
@@ -356,47 +372,52 @@ function destroy() {
...
+	 * then this method will query the wiki api again for the page details
```
Comment (Xymph) [inline Wikimate.php]: Append a period to this sentence.

**Comment by Xymph** [comment]
@waldyrious Besides the specific changes proposed above, the last WikiPage method `delete` doesn't have a Doxygen block at all. And since the code tests `$reason`'s existence, I'm thinking it should default to null in the parameter list as well: `delete( $reason = null )`

What were you saying about perfectionism again? :wink:

File: Wikimate.php [hunk 4]
```diff
@@ -130,8 +137,8 @@ public function login( $username, $password, $domain = NULL ) {
 	/**
 	 * Sets the debug mode
 	 *
-	 * @param boolean $debugMode true to turn debugging on
-	 * @return Wikimate this object
+	 * @param  boolean $debugMode true to turn debugging on
```
Comment (waldyrious) [inline Wikimate.php]: Good catch 👍

File: Wikimate.php (hunk 1 above)
Comment (waldyrious) [inline Wikimate.php]: Makes sense.

File: Wikimate.php (hunk 2 above)
Comment (waldyrious) [inline Wikimate.php]: Ah, I did catch a typo elsewhere, but this one eluded me. Will fix :)

File: Wikimate.php (hunk 3 above)
Comment (waldyrious) [inline Wikimate.php]: 👍

**Comment by waldyrious** [comment]
Great eye for detail @Xymph :) I believe I've applied all the fixes you suggested (and a couple more that I missed previously). Please take another look!

**Comment by waldyrious** [comment]
Thanks for the review 👍

**Pull request merged**